### PR TITLE
Templates: Beauty Base Animations & Links

### LINKED ISSUE #2211: Animations: Add animations to BEAUTY template to match designs
The animation process for a template is as follows:
- Applying animations in the animation editor `/story-anim-tool`
- Alter its static layout if needed to accommodate the animation
- If template requires a new animation part that doesn't already exist
  - Create ticket for missing animation part
  - Add animation part to code base
  - Go back to template and apply newly created animation part
  - Continue with template animations

Reference the figma designs:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=1%3A8

And reference the movie clips from this folder:
https://drive.google.com/drive/folders/1E0nLATcplhGNzgteOymx3s2SyA-cc7_M

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
Adds animations to the Beauty template as close to mock up as we can with existing primitives & links.

## Relevant Technical Choices
NA

## To-do
https://docs.google.com/spreadsheets/d/1iJ8Adqs4zGWteMoxVCZ2MM4LAza0LQmtPUX6I9qdshY/edit#gid=21496487

## User-facing changes
Only user facing changes should be the links being added in which shouldn't effect anything other than hovering over the link circles on page 4

## Testing Instructions
**Links**
Go to page 4 and hover over the circles with your mouse in the editor. you should see a link to `Example Domain` pop up.

**Animations**
set `enableAnimations` to `true` in `includes/Dashboard.php`. then go view the template in explore templates and see the initial pass of animations.

## Screenshots
Screen Shot 2020-07-07 at 4 28 21 PM
(Recorded at 15fps to keep file size down. to view in high fidelity, run this branch live)

Kapture 2020-07-07 at 16 31 31

---

Fixes #2211

### CODE REVIEW

**Comment by pbakaus** [comment]
Looks awesome!

File: assets/src/dashboard/templates/raw/beauty.json [hunk 1]
```diff
@@ -1368,7 +1527,12 @@
           },
           "type": "shape",
           "basedOn": "98791375-1a20-4b01-8e12-cb8c3f9932ba",
-          "id": "4b924f3a-045c-4675-9b79-383ebdd1466a"
+          "id": "4b924f3a-045c-4675-9b79-383ebdd1466a",
+          "link": {
+            "url": "http://example.com",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/beauty.json]: Minor nit, but can we use `https://` everywhere for example links?

**Review by mariano-formidable — APPROVED** [review]
The animations look great!  Before merging be sure to update the example links from `http://example.com` to `https://example.com` (as Pascal pointed out).

**Review by BrittanyIRL — APPROVED** [review]
Beautiful

**Comment by mariano-formidable** [comment]
@googlebot I consent.

**Pull request merged**